src/components/Referrals/ReferralJourney/ReferralJourney.tsx

```diff
@@ -22,13 +22,21 @@ import { ANIMATION_DURATION } from './utils'
 import { AnimationPhaseType, ReferralJourneyProps } from './ReferralJourney.types'
 
 // eslint-disable-next-line import/no-named-as-default-member
-const ReferralJourney = React.memo(({ invitedUsersAccepted }: ReferralJourneyProps) => {
+const ReferralJourney = React.memo((props: ReferralJourneyProps) => {
+  const { invitedUsersAccepted } = props
   const [animatedStep, setAnimatedStep] = useState(TIERS.filter(tier => tier.completed).length)
   const [open, setOpen] = useState(false)
   const [journeyTiers, setJourneyTiers] = useState(TIERS)
   const [animationPhase, setAnimationPhase] = useState<AnimationPhaseType>(AnimationPhaseType.WAITING_NEXT_TIER)
   const totalSteps = TIERS.length
   const journeyStepRefs = useRef<(Element | null)[]>([])
+  const sectionRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (sectionRef.current) {
+      sectionRef.current.scrollIntoView({ behavior: 'smooth', block: 'start' })
+    }
+  }, [])
```

Comment:
This effect is done here and in the `MainPage` component. Should it be in both of them?